spec_helper.rb

```diff
@@ -48,10 +48,7 @@
   end
 end
 
-dir = "../fixtures/code"
-CODE_LOADING_DIR = defined?(require_relative) ?
-                     File.realpath(dir, __FILE__) :
-                     File.expand_path(dir, __FILE__)
+CODE_LOADING_DIR = File.expand_path "../fixtures/code", __FILE__
```

Comment:
The difference is `realpath` resolves symlinks while `expand_path` won't. I am sure of the actual reason though. It comes from 6690a69a7a608f3b4e47fdef70d8776ad9a372c9.